docs/apis-tools/cli-client/index.md

```diff
@@ -39,13 +39,13 @@ zbctl [options] [command]
 ```
 
 ```
-zbctl is a command line interface designed to create and read resources inside the Zeebe broker.
-It is designed for regular maintenance jobs, such as:
-        * Deploying processes
-        * Creating jobs and process instances
-        * Activating, completing, or failing jobs
-        * Updating variables and retries
-        * Viewing cluster status
+zbctl is a command line interface designed to create and read resources inside zeebe broker.
```

Comment:
Do we know why these changes were reverted @jonathanlukas?

Reply:
No idea actually, but it seems to come from my commit...